site/content/en/docs/Reference/gameserverallocation.md

```diff
@@ -89,6 +89,7 @@ spec:
   # Defaults to all GameServers.
   # matchLabels, matchExpressions, gameServerState and player filters can be used for filtering.
   # See: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/ for more details on label selectors.
+  # Desprecated, use field selectors instead.
```

Comment:
nit: spelling "Deprecated"

Reply:
These changes also need to be guarded so that they don't go live on the website until the next release.

Reply:
It does only publish on 1.17 with {{% feature publishVersion="1.17.0" %}} which somebody already added.

Reply:
Thanks, the diff hid the release guard.

Lines 93, 177, & 179 still have the mis-spelling of Deprecated (although it's been fixed elsewhere).

Reply:
Done.